src/tests/unittests/util/testimg.h

```diff
@@ -0,0 +1,162 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2020 Martin Burri.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ * Test image generation, access, printing etc. to be used for unit testing with
+ * cmocka.
+ *
+ * Please see ../README.md for more detailed documentation.
+ */
+
+typedef struct Testimg {
```

Comment:
current style is to put { on next line.

Reply:
Oh yes, missed that one. Thanks for fixing and merging!